src/core/Radio/index.stories.tsx

```diff
@@ -1,77 +1,117 @@
-import { FormControlLabel } from "@mui/material";
-import { action } from "@storybook/addon-actions";
-import { storiesOf } from "@storybook/react";
+import { FormControlLabel, RadioGroup } from "@mui/material";
+import { Args, Story } from "@storybook/react";
 import React from "react";
 import RadioButton from "./index";
 
-const actions = {
-  onChange: action("onChange"),
+const DefaultDemo = (props: Args): JSX.Element => {
+  const { text } = props;
+
+  return (
+    <RadioGroup
+      aria-labelledby="demo-radio-buttons-group-label"
+      defaultValue="demo"
+      name="radio-buttons-group"
+    >
+      <FormControlLabel
```

Comment:
It is a good practice to pass the `{...props}` to the component so that the storybook controls could change the component! 
Currently, one can't disable the radio input for example!